client/components/data/query-atat-eligibility/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { requestEligibility } from 'state/automated-transfer/actions';
+
+export class QueryAutomatedTransferEligibility extends Component {
+	static propTypes = {
+		request: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
```

Comment:
Do we need to be so strict for siteId? For a falsey value we can just noop. It makes it easier to use generally since the caller can pass through a this.props.siteId even when it's still loading the value.